CordovaLib/Cordova/Cordova.h

```diff
@@ -0,0 +1,19 @@
+//
+//  Cordova.h
+//  Cordova
+//
+//  Created by Arnold Goldberg on 1/25/17.
+//
+//
+
+#import <UIKit/UIKit.h>
+
+//! Project version number for Cordova.
+FOUNDATION_EXPORT double CordovaVersionNumber;
+
+//! Project version string for Cordova.
+FOUNDATION_EXPORT const unsigned char CordovaVersionString[];
+
+// In this header, you should import all the public headers of your framework using statements like #import <Cordova/PublicHeader.h>
```

Comment:
All of the public headers should be represented in the umbrella header, a la:

`#import <Cordova/CDVViewController.h`

And so on.